.nvmrc

```diff
@@ -1 +1 @@
-8.17.0
\ No newline at end of file
+13.14.0
```

Comment:
Node recommends we should be using the [active LTS version](https://nodejs.org/en/about/releases/), 12.18.0.

Reply:
I'm juggling with different versions of circle ci node-browser images and node versions, so I might not be able to stay in LTS